include/glow/Graph/Graph.h

```diff
@@ -276,9 +276,8 @@ class Function final : public Named {
   /// is the constant weights and will be row-wise quantized during node
   /// creation time.
   RowwiseQuantizedFullyConnectedNode *
-  createRowwiseQuantizedFullyConnected(Context &ctx, llvm::StringRef name,
-                                       NodeValue input, Variable *W, Node *B,
-                                       TypeRef outTy);
+  createRowwiseQuantizedFullyConnected(llvm::StringRef name, NodeValue input,
```

Comment:
can you also mention in the doxygen the outtype expectations. That's it's quantized in a regular way etc.